config/software/netuitive-diamond.rb

```diff
@@ -1,5 +1,5 @@
 name "netuitive-diamond"
-default_version "master"
+default_version "bugfix/concurrentlog"
```

Comment:
Will you merge this PR with this branch referenced as part of the staged rollout?

Reply:
Switched back to master in 6008008. My thinking was each of these PR's could be merged into dev and master, without conflict, as long as they were done subsequently.  Leaving the default_version pointed to the bugfix branch in netuitive-diamond.rb was a mistake.